### ISSUE TITLE
[Item] Ethereal Essence Shere doesn't land in target's bag.

### ISSUE THREAD

**Issue by Linozo** [dalaranwow/dalaran-wow#4395]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Doesn't land in target's bag.

**Expected behaviour**: Should end up in target's bag.
http://www.wowhead.com/item=38308/ethereal-essence-sphere

**Steps to reproduce the problem**:

1.  Throw this item to someone.
2. 
3. 

**Include proofs for this behaviour**
I've voted for it once, but now I can't anymore since the NPC is bugged.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/1249, the fix must be tested on PTR first
